query/stdlib/influxdata/influxdb/rules_test.go

```diff
@@ -1995,25 +1957,9 @@ func TestPushDownBareAggregateRule(t *testing.T) {
 		}
 	}
 
-	minProcedureSpec := func() *universe.MinProcedureSpec {
-		return &universe.MinProcedureSpec{
-			SelectorConfig: execute.SelectorConfig{Column: "_value"},
-		}
-	}
-	countProcedureSpec := func() *universe.CountProcedureSpec {
-		return &universe.CountProcedureSpec{
-			AggregateConfig: execute.AggregateConfig{Columns: []string{"_value"}},
-		}
-	}
-	sumProcedureSpec := func() *universe.SumProcedureSpec {
-		return &universe.SumProcedureSpec{
-			AggregateConfig: execute.AggregateConfig{Columns: []string{"_value"}},
-		}
-	}
-
 	testcases := []plantest.RuleTestCase{
 		{
-			// successful push down
+			// successful push down count()
```

Comment:
Can you replace these types of comment with ones of the form `ReadRange -> first => ReadWindowAggregate`?